challenges/objects-arrays.js

```diff
@@ -98,28 +139,39 @@ The zoos need a list of all their animal's names (names only, not scientific) co
 */
 
 const lowerCase = [];
+  zooAnimals.map(element => {
+    lowerCase.push(element.animal_name.toLowerCase())
```

Comment:
Because `.map()` returns a new array without mutating the original, we can refactor this to:
```const lowerCase = zooAnimals.map(element => element.animal_name.toLowerCase())```